src/utility/copy.cpp

```diff
@@ -118,48 +108,50 @@ CheckQueryPermissions(Query *query, const char *query_string) {
 	}
 }
 
-static duckdb::string
-CreateCopyOptions(CopyStmt *copy_stmt, bool *options_valid) {
+static void
+appendCreateCopyOptions(StringInfo info, CopyStmt *copy_stmt) {
 	if (list_length(copy_stmt->options) == 0) {
-		return ";";
+		appendStringInfo(info, ";");
+		return;
 	}
 
-	std::ostringstream options_stream;
-	options_stream << "(";
+	appendStringInfo(info, "(");
 
 	bool first = true;
 	foreach_node(DefElem, defel, copy_stmt->options) {
-		if (!first) {
-			options_stream << ", ";
+		if (first) {
+			first = false;
+		} else {
+			appendStringInfo(info, ", ");
 		}
-		options_stream << defel->defname;
+
+		appendStringInfoString(info, defel->defname);
 		if (defel->arg) {
-			options_stream << " ";
+			appendStringInfo(info, " ");
 			switch (nodeTag(defel->arg)) {
 			case T_Integer:
 			case T_Float:
 			case T_Boolean:
-				options_stream << defGetString(defel);
+				appendStringInfoString(info, defGetString(defel));
 				break;
 			case T_String:
 			case T_TypeName:
-				options_stream << quote_literal_cstr(defGetString(defel));
+				appendStringInfoString(info, quote_literal_cstr(defGetString(defel)));
 				break;
 			case T_List:
-				options_stream << NameListToQuotedString((List *)defel->arg);
+				appendStringInfoString(info, NameListToQuotedString((List *)defel->arg));
 				break;
 			case T_A_Star:
-				options_stream << "*";
+				appendStringInfo(info, "*");
 				break;
 			default:
-				throw std::runtime_error("unexpected node type in COPY: " + std::to_string((int)nodeTag(defel->arg)));
+				// elog(ERROR, "Expected single table to be created, but found %" PRIu64, static_cast<uint64_t>(SPI_processed));
```

Comment:
Is this commented out line here on purpose?

Reply:
Nope, a leftover... Thanks!